lib/modules/manager/pre-commit/extract.ts

```diff
@@ -79,15 +81,20 @@ function determineDatasource(
 function extractDependency(
   tag: string,
   repository: string,
-): {
-  depName?: string;
-  depType?: string;
-  datasource?: string;
-  packageName?: string;
-  skipReason?: SkipReason;
-  currentValue?: string;
-} {
-  logger.debug(`Found version ${tag}`);
+  comment?: string,
+): PackageDependency {
+  let currentValue = tag;
+  let currentDigest: string | undefined;
+
+  if (tag.length === 40 && comment) {
+    const match = /.*frozen: (.+)$/.exec(comment);
```

Comment:
import and use `regEx` wrapper here
```suggestion
    const match = regEx(/.*frozen: (.+)$/).exec(comment);
```